tools/monorepo-utils/package.json

```diff
@@ -0,0 +1,35 @@
+{
+	"name": "monorepo-utils",
+	"version": "0.0.1",
+	"description": "WooCommerce Monorepo utility tooling.",
+	"author": "Automattic",
+	"homepage": "https://github.com/woocommerce/woocommerce",
+	"license": "GPLv2",
+	"repository": "woocommerce/woocommerce",
+	"dependencies": {
+		"chalk": "^4.1.2",
+		"commander": "^9.4.0",
+		"dotenv": "^10.0.0",
+		"ts-jest": "^29.1.0"
```

Comment:
Another dev dependency!

Reply:
This one is actually needed to run jest directly on TS files.

Reply:
@psealock but thats a dev task to run jest right? whence a dev dependency